client/plots/profilePlot.js

```diff
@@ -24,13 +24,14 @@ export class profilePlot {
 	getState(appState) {
 		const config = appState.plots.find(p => p.id === this.id)
 		if (!config) throw `No plot with id='${this.id}' found`
+		const [logged, site, user] = getProfileLogin() //later on replace with real login info
 		return {
 			config,
 			termfilter: appState.termfilter,
 			dslabel: appState.vocab.dslabel,
 			vocab: appState.vocab,
-			logged: true, //later change to read login info
-			site: config.site,
+			logged: logged, //later change to read login info
```

Comment:
Could use `logged,` syntax shortcut, no need for `logged: logged,`, similar to using the `site,` shortcut.

Reply:
Done